.docker/docker-compose.dev.yml

```diff
@@ -0,0 +1,32 @@
+name: scancontrol
+
+services:
+  # Builds the ROS package
+  scancontrol-build:
+    container_name: scancontrol-build
+    build:
+      context: ${SCANCONTROL_SRC_DIR}
+      dockerfile: .docker/build.Dockerfile
+      args:
+        - TARGET_WS=${SCANCONTROL_TARGET_WS}
+    network_mode: host 
+    image: "scancontrol:build"
```

Comment:
This should probably be removed, since this is suppose to create that image.

Reply:
This is here to tag the name of the image that the scancontrol-service will be using

Reply:
Isn't that what `container_name` does?